# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1334102275 -43200
# Node ID 1aafe9e68ee5e4f223d240da323a5e33e64e8a07
# Parent  7d8ac0ee679de0e5eeba4966fe280f5f65295472
change path to stdsage.pxi in degree_sequence.pyx

diff --git a/sage/combinat/degree_sequences.pyx b/sage/combinat/degree_sequences.pyx
--- a/sage/combinat/degree_sequences.pyx
+++ b/sage/combinat/degree_sequences.pyx
@@ -262,7 +262,7 @@
 from sage.libs.gmp.all cimport mpz_t
 from sage.libs.gmp.all cimport *
 from sage.rings.integer cimport Integer
-include '../../../../devel/sage/sage/ext/stdsage.pxi'
+include '../ext/stdsage.pxi'
 include '../ext/cdefs.pxi'
 include "../ext/interrupt.pxi"
 
